src/components/FormRow/FormRow.scss

```diff
@@ -0,0 +1,49 @@
+@use '../variables';
+
+.#{variables.$ns}form-row {
+    --yc-form-row-label-width: 172px;
+    --yc-form-row-field-height: 28px;
```

Comment:
I think it should have `gc-` preffix, like [ns](https://github.com/gravity-ui/components/blob/main/src/components/utils/cn.ts#L3)